### ISSUE TITLE
Some genealogy of classes not being returned 

### ISSUE THREAD

**Issue by YaphetKG** [biolink/biolink-model-toolkit#78]
When running this code snippet 

```python 

!pip install git+https://github.com/biolink/biolink-model-toolkit.git@0.8.2
version = "2.2.16"
BMT = Toolkit(
            schema=f"[https://raw.githubusercontent.com/biolink/biolink-model/{version}/biolink-model.yaml](https://raw.githubusercontent.com/biolink/biolink-model/%7Bversion%7D/biolink-model.yaml)",
        )
classes = BMT.get_all_classes()
# get_all_slots()
slots = BMT.get_all_slots()
# get_all_types()
types = BMT.get_all_types()
# get_all_elements()
elements = classes + slots  + types
basic_ancestors = {
    element: BMT.get_ancestors(
        element,
        reflexive=False,
        mixin=False,
    )
    for element in elements
}
basic_ancestors['expressed in']

```

Although `expressed in` is in the [biolink heirarchy as a subclass of](https://github.com/biolink/biolink-model/blob/master/biolink-model.yaml#L4901-L4918) `located in` we don't get the   located in the the results and calling BMT.get_descendants("located in") also doesn't report back `expressed in`

**Comment by cbizon** [comment]
@sierra-moxon

**Comment by YaphetKG** [comment]
Issue seems to be fixed on biolink model v2.3.0
